k8s.gcr.io/ensure-prod-repo.sh

```diff
@@ -35,57 +35,67 @@ if [ $# != 0 ]; then
 fi
 
 # The GCP project names.
+TEST_PROJECT="k8s-cip-test-prod"
 PROD_PROJECT="k8s-gcr-prod"
 TRASH_PROJECT="k8s-gcr-graveyard"
 
+ALL_PROJECTS="${TEST_PROJECT} ${PROD_PROJECT} ${TRASH_PROJECT}"
+
 # Regions for prod.
 PROD_REGIONS=(us eu asia)
 
 # Make the projects, if needed
-color 6 "Ensuring prod project exists: ${PROD_PROJECT}"
-ensure_project "${PROD_PROJECT}"
-color 6 "Ensuring graveyard project exists: ${TRASH_PROJECT}"
-ensure_project "${TRASH_PROJECT}"
+for prj in ${ALL_PROJECTS}; do
```

Comment:
Why not just use 1 loop `for prj in ${ALL_PROJECTS}; do` and move each component into it (instead of wrapping each component with a loop)?

I think having 1 loop would be easier in case this script dies (you wouldn't have to check any of the projects that have not yet been iterated through by the for-loop).

Reply:
Easier as in easier to debug, that is.

Reply:
Indeed that would be simpler.  Myopia...

Will push with that change momentarily.